src/BenchmarksApps/TodosApi/TodosApi.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <Nullable>enable</Nullable>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <ServerGarbageCollection>false</ServerGarbageCollection>
+    <InvariantGlobalization>true</InvariantGlobalization>
+    <UserSecretsId>b8ffb8d3-b768-460b-ac1f-ef267c954c85</UserSecretsId>
+    <PublishAot>true</PublishAot>
+    <EnableLogging Condition="$(Configuration.StartsWith('Debug'))">true</EnableLogging>
+    <DefineConstants Condition=" '$(EnableLogging)' == 'true' ">$(DefineConstants);ENABLE_LOGGING</DefineConstants>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Authentication.JwtBearer" Version="8.0.0-preview.3.23177.8" />
```

Comment:
is there a way to keep this version up to date with daily builds?

cc @sebastienros

Reply:
We can use a wildcard, e.g. `8.0.0-preview.*`?

Reply:
Seems there's a `$(MicrosoftAspNetCoreAppPackageVersion)` property defined, I'll try using that.

Reply:
Oh it's hard-coded to a preview.1 version 😱

Reply:
I think that might get replaced by crank.... @sebastienros ?

Reply:
Yeah I've noticed the projects aren't always set up for ease of local use so I made some tweaks to improve that as part of this PR now, i.e. ensuring there are appropriate package versions set.

Reply:
You can have a default version set for local development, and fallback to $(MicrosoftAspNetCoreAppPackageVersion) if it's defined. Crank will set it on builds. There is the same one for the runtime.

Reply:
I humbly suggest we shouldn't be relying on anything local being manually configured to get a good dev experience. The solution/projects should be openable locally after cloning without getting warnings.

Reply:
@sebastienros are you suggesting we should change all the package version references in the app project files to hard-code a default version (can include a wildcard) but prefer `$(MicrosoftAspNetCoreAppPackageVersion)` if it's set? It's a shame we can't easily make that more elegant in MSBuild but it will certainly achieve what I think we're looking for.

Reply:
I think you can keep have something like 
```
<MicrosoftAspNetCoreAuthenticationJwtBearerVersion>8.0.0-preview.3.23177.8</MicrosoftAspNetCoreAuthenticationJwtBearerVersion>
<MicrosoftAspNetCoreAuthenticationJwtBearerVersion Condition="'$(MicrosoftAspNetCoreAppPackageVersion)' != ''">$(MicrosoftAspNetCoreAppPackageVersion)</MicrosoftAspNetCoreAuthenticationJwtBearerVersion>

```

This way it will build locally, but use the version aligned with aspnet when using crank.